### ISSUE TITLE
Integrations: deprecate old incoming webhook URLs

### ISSUE THREAD

**Issue by stsewd** [readthedocs/readthedocs.org#11037]
## What's the problem this feature will solve?

Currently, we have two URLs for incoming webhooks

https://github.com/readthedocs/readthedocs.org/blob/e5143773fc0826515f5bf508d4b659d8e9a4303f/readthedocs/api/v2/urls.py#L76-L105

One that only takes into consideration the project's slug, which means that only one integration of a type can exist per project.
And the other one take into consideration the project and integration PK, so many integrations of the same type can co-exist.

Looks like we are currently linking to the more specific type of URL for all integrations, any project using the old URL is probably an old project.

## Describe the solution you'd like

Just use one URL, ask users to migrate to the more specific URL type.

Maybe it also makes sense to allow only one incoming webhook, not sure when multiple incoming webhooks would be necessary, maybe for debugging only?

## Additional context

Currently, if the user makes use of the more specific URL, and has more than one type of that integration, the integration will fail (since we use .get(), and it returns more than one result)

https://read-the-docs.sentry.io/issues/4634041415/?project=148442&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=24h&stream_index=3

**Comment by humitos** [comment]
Related to https://github.com/readthedocs/readthedocs.org/issues/10760

**Comment by agjohnson** [comment]
Needed here:

- [ ] Get number of users hitting these URLs
- [ ] Plan a deprecation

**Comment by ericholscher** [comment]
Interestingly the Sentry issue here isn't found, so wonder if we fixed that particular issue?

**Comment by ericholscher** [comment]
Nope, looks like it's this one now: https://read-the-docs.sentry.io/issues/4951512277 -- but that's on the deprecation warning for something else: https://blog.readthedocs.com/security-update-on-incoming-webhooks/.
